zaas-service/src/main/java/org/zowe/apiml/zaas/health/ZaasHealthIndicator.java

```diff
@@ -67,19 +51,10 @@ protected void doHealthCheck(Health.Builder builder) {
 
         int zaasCount = this.discoveryClient.getInstances(CoreService.ZAAS.getServiceId()).size();
 
-        builder.status(toStatus(discoveryUp))
-            .withDetail(CoreService.DISCOVERY.getServiceId(), toStatus(discoveryUp).getCode())
+        builder.status(toStatus(authUp))
             .withDetail(CoreService.AUTH.getServiceId(), toStatus(authUp).getCode())
             .withDetail("zaasCount", zaasCount);
```

Comment:
does zaas need to know about other zaas instances? I guess this is still from the gateway logic that should check whether the authentication is complete up

Reply:
shouldn't be valid for zaas too? as it is eventually used for the authentication itself